### DISCUSSION TITLE
汇总表中数据存在相同维度，指标不同的情况，使用2.0.0 版本vue组件生成时，维度相同的数据会被过滤掉，有什么办法可以显示相同维度，不同指标的所有数据

### DISCUSSION THREAD

**Discussion by caiyan1121** [antvis/S2#2912] [Q&A]
![企业微信截图_20240927173244](https://github.com/user-attachments/assets/46f1a383-cd15-48b3-a242-0561a385fe2a)

**Comment by lijinke666** [comment]
可以参考 https://github.com/antvis/S2/issues/2767#issuecomment-2164209195
